RELEASE_NOTES.txt

```diff
@@ -1,6 +1,138 @@
 Apache Ignite Release Notes
 ===========================
 
+Apache Ignite In-Memory Distributed Database 2.16.0
+-----------------------------------------------------------
+
+(!) WARNINGS:
+* Operations on atomic caches within transactions are finally forbidden. The system property IGNITE_ALLOW_ATOMIC_OPS_IN_TX has been removed.
+* Removed CacheAtomicityMode#TRANSACTIONAL_SNAPSHOT cache mode.
+* Mixed atomicity cache groups now restricted, but IGNITE_ALLOW_MIXED_CACHE_GROUPS system option may temporarily allow them.
+* The ignite-ml, cassandra modules have been migrated to the Ignite extensions.
+
+Ignite:
+* Added ARM64 Docker images.
+* Added CLIENT_CONNECTION_ATTRIBUTES system view.
+* Added IndexQuery to performance statistics.
+* Added JMX commands invoker.
+* Added a description check if the command argument is enum type.
+* Added ability to configure metric exporter filter from Spring XML.
+* Added cache dumps.
+* Added client session outbound message queue size limit.
+* Added compact footer usage statistics to the idle_verify command.
+* Added descriptions and values to the properties list command.
+* Added documentation for the Index Reader Utility.
```

Comment:
Do we really need documentation tickets in release notes? Documentation even not in release package.

Reply:
Done.